### PR TITLE
Feature/asb 4 5 7

### PR DESCRIPTION
**Pull request by lvnhmd**

### CODE REVIEW

File: packages/forms-web-app/src/views/having-your-say-guide/registering-have-your-say.njk [hunk 1]
```diff
@@ -40,40 +40,52 @@
...
+                
+                <h2 class="govuk-heading-m" id="after-you-have-registered">After you have registered</h2>
+                <p class="govuk-body">Once you have registered to have your say, your name and comments will be published on the project section of this website. Once the registration period closes, you can see all of the other registration comments (also known as Relevant Representations) from other people.</p>
+                <p class="govuk-body">Once the Examination of the application starts, you will be able to submit more information at the deadlines in the timetable. You can do this using the online submission form in the specific project section of this website.</p>
+
+                <h2 class="govuk-heading-m" id="register-to-have-your-say">Register to have your say</h2>
+                <p class="govuk-body">The deadline for registering will be in the developer's advert, or you can check the project page.</p>
+                <p class="govuk-body"><a href="../examination/{{ caseRef }}">Go to the {{ projectName }} overview</a></p>
```
Comment (pk2453) [inline packages/forms-web-app/src/views/having-your-say-guide/registering-have-your-say.njk:81]: I think caseRef and projectName are not available dynamically. So either requirement should be to hardcode both depending on which project they want.

**Review by pk2453 — COMMENTED** [review]
Check with design about overview link on http://localhost:9004/having-your-say-guide/registering-have-your-say

File: packages/forms-web-app/src/views/having-your-say-guide/registering-have-your-say.njk (hunk 1 above)
Comment (pk2453) [reply packages/forms-web-app/src/views/having-your-say-guide/registering-have-your-say.njk:81]: Also branch has conflicts.

**Comment by lvnhmd** [comment]
> Check with design about overview link on http://localhost:9004/having-your-say-guide/registering-have-your-say

We have been through refinement and this is what we agreed.

**Pull request merged**